Gemfile.lock

```diff
@@ -243,7 +243,7 @@ GEM
     mini_portile2 (2.4.0)
     minitest (5.11.3)
     mono_logger (1.1.0)
-    multi_json (1.12.1)
+    multi_json (1.13.1)
```

Comment:
Some of these gems are not just used in test, so I'm gonna remove the Tests Only label for now: multi_json, rack, rails-html-sanitizer, rake.

Reply:
Good catch, thank you!

I've added dump.rdb to my own git ignore, since I seem to be the only one having this issue of it sneaking in.

Reply:
This PR will collect conflicts fast. Is it possible to update the test-only gems but not the rest?

Reply:
I don't know! I didn't update them on purpose, the updates were pulled in by the gems that I intentionally updated, so they must be declared as dependencies.